app/packages/state/src/hooks/useSchemaSettings.ts

```diff
@@ -70,11 +72,17 @@ const disabledField = (
   groupField?: string
 ) => {
   const currField = combinedSchema?.[path] || ({} as Field);
-  const { ftype } = currField;
+  const { ftype, embeddedDocType } = currField;
 
   const parentPath = path.substring(0, path.lastIndexOf("."));
   const parentField = combinedSchema?.[parentPath];
   const parentEmbeddedDocType = parentField?.embeddedDocType;
+  const pathSplit = path.split(".");
+  const embeddedDocTypeSplit = embeddedDocType?.split(".");
+  const hasDynamicEmbeddedDocument = [
+    DYNAMIC_EMBEDDED_DOCUMENT_FIELD,
```

Comment:
Do we still need DYNAMIC_EMBEDDED_DOCUMENT_FIELD? Don't think we are still using dynamic embedded document as a field.

Reply:
just to confirm you are saying we don't need to check for the first one anymore?
```
export const DYNAMIC_EMBEDDED_DOCUMENT_FIELD =
  "fiftyone.core.fields.DynamicEmbeddedDocumentField";
export const DYNAMIC_EMBEDDED_DOCUMENT_FIELD_V2 =
  "fiftyone.core.odm.embedded_document.DynamicEmbeddedDocument";
```